### PR TITLE
Navigation and a bit of clean up

### PR DESCRIPTION
**Pull request by scribblecrumb**
Enabled 'back to top' button and had some fun playing around with the features in some of the art files.  Also moved the style guides out of the sub-directory, as we don't really need them in a sub-directory atm.

Removes awesome-pages and implements section indexing.

### CODE REVIEW

File: .cache/plugin/git-committers/page-authors.json
Comment (archanyhm) [inline .cache/plugin/git-committers/page-authors.json:1]: Please remove this file from this branch via rebase.

File: .idea/workspace.xml
Comment (archanyhm) [inline .idea/workspace.xml:1]: Please remove this file via rebase.

**Review by archanyhm — CHANGES_REQUESTED** [review]
See below

**Comment by scribblecrumb** [comment]
> See below

I think? I managed to do that correctly

File: docs/Art/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Basic Information
+[Camp Backgrounds](https://clangen.io/docs/Art/camp-backgrounds/){.md-button}
+[Cat Sprites](https://clangen.io/docs/Art/cat-sprites/){.md-button}
+[Clan Symbols](https://clangen.io/docs/Art/clan_symbols/){.md-button}
+[Patrol Art](https://clangen.io/docs/Art/patrol-sprites/){.md-button}
```
Comment (archanyhm) [inline docs/Art/index.md]: Last but not least: please change those links to be in the proper format -> relative links to the .md file. MkDocs will automatically adjust the links.

Comment (scribblecrumb) [reply docs/Art/index.md]: Ah gotcha! Fixed

**Comment by archanyhm** [comment]
I also tested this locally and most links are broken.

```
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
             - index.md
             - code/index.md
             - code/content-adding.md
             - project-basics/development-team-roles.md
             - project-basics/getting-started-with-github-and-clangen.md
             - writing/index.md
             - writing/advanced-documentation.md
             - writing/leader-ceremonies.md
             - writing/leaders-den-events.md
             - writing/patrols.md
             - writing/shortevents.md
             - writing/thoughts.md
WARNING -  A reference to 'Project basics/development-team-roles.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Project basics/getting-started-with-github-and-clangen.md' is included in the 'nav' configuration, which is not found in the documentation
           files.
WARNING -  A reference to 'Code/index.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Code/content-adding.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Writing/index.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Writing/patrols.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Writing/shortevents.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Writing/leaders-den-events.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Writing/thoughts.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Writing/leader-ceremonies.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  A reference to 'Writing/advanced-documentation.md' is included in the 'nav' configuration, which is not found in the documentation files.
WARNING -  Doc file 'index.md' contains a link 'art/basic.md', but the target is not found among documentation files.
WARNING -  Doc file 'index.md' contains a link 'writing/basic.md', but the target is not found among documentation files.
WARNING -  Doc file 'index.md' contains a link 'code/basic.md', but the target is not found among documentation files.
WARNING -  Doc file 'index.md' contains a link 'audio/basic.md', but the target is not found among documentation files.
WARNING -  [git-revision-date-localized-plugin] '/Users/philippmicke/Developer/clangen/clangen-docs/docs/code/index.md' has no git logs, using current timestamp
WARNING -  [git-revision-date-localized-plugin] '/Users/philippmicke/Developer/clangen/clangen-docs/docs/code/index.md' has no git logs, using current timestamp
WARNING -  Doc file 'writing/leaders-den-events.md' contains a link 'basic.md#statuses', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/leaders-den-events.md' contains a link 'basic.md#ages', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/leaders-den-events.md' contains a link 'basic.md#traits', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/leaders-den-events.md' contains a link 'basic.md#skills', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/leaders-den-events.md' contains a link 'basic.md#ages', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/leaders-den-events.md' contains a link 'basic.md#traits', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/leaders-den-events.md' contains a link 'basic.md#skills', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/patrols.md' contains a link 'basic.md#the-basics', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/patrols.md' contains a link 'basic.md#coding-terms-for-writers-to-know', but the target 'writing/basic.md' is not found among documentation
           files.
WARNING -  Doc file 'writing/patrols.md' contains a link 'basic.md#clangen-biomes', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/shortevents.md' contains a link 'basic.md#clangen-biomes', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/shortevents.md' contains a link 'basic.md#ages', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/shortevents.md' contains a link 'basic.md#statuses', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/shortevents.md' contains a link 'basic.md#injuries', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/shortevents.md' contains a link 'basic.md#scars', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/shortevents.md' contains a link 'basic.md#writing-histories', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#illnesses', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#injuries', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#permanent-conditions', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#backstories', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#statuses', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#ages', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#traits', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#skills', but the target 'writing/basic.md' is not found among documentation files.
WARNING -  Doc file 'writing/thoughts.md' contains a link 'basic.md#other-statuses', but the target 'writing/basic.md' is not found among documentation files.
```

**Pull request merged**